docker-compose.elastic.logging.yml

```diff
@@ -1,4 +1,4 @@
-version: "2.1"
+version: "3.8" # Docker Engine minimum version 19.03.0
```

Comment:
could you check if it's working?
had an issue with that https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/163

Reply:
Yes, I have been using this branch with
* Docker Compose version v2.12.2
* Docker version 20.10.21